packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -35,6 +36,14 @@ type MultiFileFormFieldConfig = { type: "fileUpload"; multiple: true; maxFiles?:
 export type FormFieldConfig<T> = (
     | { type: "text"; multiline?: boolean }
     | { type: "number" }
+    | {
+          type: "numberRange";
+          minValue: number;
+          maxValue: number;
+          disableSlider?: boolean;
+          startAdornment?: ReactNode;
+          endAdornment?: ReactNode;
```

Comment:
Should we add adornment support as a general feature for all field types?

Reply:
Why not, especially for number input

For normal text input i dont see a use case, but the config option also wont do any harm

Reply:
Should I add that here or is that something you'd rather want in its own PR?

Reply:
Please do this in a follow-up PR